ci-operator/config/Azure/ARO-HCP/Azure-ARO-HCP-main__periodic.yaml

```
@@ -69,57 +107,87 @@ tests:
     - ref: aro-hcp-provision-generate-resource-group-name
     test:
     - chain: aro-hcp-provision-create-functional-cluster
-- as: integration-e2e-parallel
+- as: prod-uksouth-e2e-parallel
   cron: 0 */4 * * *
   steps:
     env:
-      ARO_HCP_SUITE_NAME: integration/parallel
-      CUSTOMER_SUBSCRIPTION: ARO SRE Team - INT (EA Subscription 3)
+      ARO_HCP_SUITE_NAME: prod/parallel
+      CUSTOMER_SUBSCRIPTION: ARO Hosted Control Planes (EA Subscription 1)
       LOCATION: uksouth
     post:
     - ref: aro-hcp-provision-aro-hcp-gather-extra
     - chain: aro-hcp-provision-teardown-cluster
     test:
     - ref: aro-hcp-tests-run-aro-hcp-tests
-- as: stage-e2e-parallel
+- as: delete-expired-prod-uksouth-resource-groups
+  cron: 7,37 * * * *
+  steps:
+    env:
+      CUSTOMER_SUBSCRIPTION: ARO Hosted Control Planes (EA Subscription 1)
+    test:
+    - ref: aro-hcp-provision-delete-all-expired-resource-groups
+- as: create-aro-hcp-in-prod-brazilsouth
   cron: 0 */4 * * *
   steps:
     env:
-      ARO_HCP_SUITE_NAME: stage/parallel
-      CUSTOMER_SUBSCRIPTION: ARO HCP - STAGE testing (EA Subscription)
-      LOCATION: uksouth
+      CLUSTER_NAME: periodic-prod-brazilsouth-cluster
+      CUSTOMER_RESOURCE_GROUP_PREFIX: periodic-prod-brazilsouth-customer-rg
+      LOCATION: brazilsouth
+      SUBSCRIPTION: ARO Hosted Control Planes (EA Subscription 1)
     post:
     - ref: aro-hcp-provision-aro-hcp-gather-extra
     - chain: aro-hcp-provision-teardown-cluster
+    pre:
+    - ref: aro-hcp-provision-generate-resource-group-name
     test:
-    - ref: aro-hcp-tests-run-aro-hcp-tests
-- as: prod-e2e-parallel
+    - chain: aro-hcp-provision-create-functional-cluster
+- as: prod-brazilsouth-e2e-parallel
   cron: 0 */4 * * *
```

```suggestion
  cron: '@daily'
```

example here: https://github.com/openshift/release/blob/7951c809a5d92e8128eaa9a06066e6b87a1c21f3/ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-netobserv-perf-tests__netobserv-aws-4.21-nightly-x86.yaml#L35

new region testing doesn't need to be as frequent.